scripts/build.sh

```diff
@@ -38,6 +38,16 @@ find ./build -type f -name "logo.png" | while read logo; do
   fi
 done
 
+# Create fallback for dark variants
+find ./build -type f -type f -name "icon.png" -o -name "icon@2x.png" -o -name "logo.png" -o -name "logo@2x.png" | while read image; do
+  dir=$(dirname "${image}")
+  filename=$(basename -s .png "${image}")
+  if [[ ! -f "${dir}/dark_${filename}.png" ]]; then
+    cp "${image}" "${dir}/dark_${filename}.png"
```

Comment:
Not sure if we should or if it would be better, but couldn't we use rewrites instead of copying the images?

Reply:
I guess we could?